src/linter.rs

```diff
@@ -285,32 +313,33 @@ impl Linter {
     filtered_diagnostics
   }
 
-  fn lint_module(
+  fn lint_program(
     &self,
     file_name: String,
-    module: swc_ecmascript::ast::Module,
+    program: swc_ecmascript::ast::Program,
     comments: SingleThreadedComments,
   ) -> Vec<LintDiagnostic> {
     let start = Instant::now();
-    let file_ignore_directive = comments.with_leading(module.span.lo(), |c| {
-      let directives = c
-        .iter()
-        .filter_map(|comment| {
-          parse_ignore_comment(
-            &self.ignore_file_directives,
-            &*self.ast_parser.source_map,
-            comment,
-            true,
-          )
-        })
-        .collect::<Vec<IgnoreDirective>>();
-
-      if directives.is_empty() {
-        None
-      } else {
-        Some(directives[0].clone())
-      }
-    });
+    let file_ignore_directive =
+      comments.with_leading(program.span().lo(), |c| {
+        let directives = c
+          .iter()
+          .filter_map(|comment| {
+            parse_ignore_comment(
+              &self.ignore_file_directives,
+              &*self.ast_parser.source_map,
+              comment,
+              true,
+            )
+          })
+          .collect::<Vec<IgnoreDirective>>();
+
+        if directives.is_empty() {
+          None
+        } else {
+          Some(directives[0].clone())
+        }
+      });
```

Comment:
```suggestion
      comments.with_leading(program.span().lo(), |c| {
        c.iter().find_map(|comment| {
          parse_ignore_comment(
            &self.ignore_file_directives,
            &*self.ast_parser.source_map,
            comment,
            true,
          )
        })
      });
```

Unrelated to the essence of this PR, but we could make it simple like this with better performance 😃